TheWarWithin/Priorities/DeathKnightBlood.simc

```diff
@@ -1,90 +1,82 @@
+actions.deathbringer+=/death_and_decay,if=!buff.death_and_decay.up
+actions.deathbringer+=/marrowrend,if=buff.exterminate.up|(buff.bone_shield.stack<5&!dot.bonestorm.ticking)
 actions.deathbringer+=/death_strike
+actions.deathbringer+=/tombstone,if=buff.bone_shield.stack>=8&buff.death_and_decay.remains&cooldown.dancing_rune_weapon.remains>=25
+actions.deathbringer+=/blooddrinker,if=!buff.dancing_rune_weapon.up&active_enemies<=2&buff.coagulopathy.remains>3
 actions.deathbringer+=/consumption
-actions.deathbringer+=/blood_boil,if=charges_fractional>=1.5
-actions.deathbringer+=/heart_strike,if=rune>=1|rune.time_to_2<gcd
 actions.deathbringer+=/blood_boil
+actions.deathbringer+=/heart_strike,if=buff.coagulopathy.stack<5
 actions.deathbringer+=/heart_strike
+actions.deathbringer+=/soul_reaper,if=buff.reaper_of_souls.up
 actions.deathbringer+=/arcane_torrent,if=runic_power.deficit>20
-actions.deathbringer+=/deaths_caress,if=buff.bone_shield.stack<11
-
-actions.high_prio_actions=blood_tap,use_off_gcd=1,if=(rune<=2&rune.time_to_3>gcd.max&charges_fractional>=1.8)
-actions.high_prio_actions+=/blood_tap,use_off_gcd=1,if=(rune<=1&rune.time_to_3>gcd.max)
-actions.high_prio_actions+=/raise_dead,use_off_gcd=1
-actions.high_prio_actions+=/deaths_caress,if=buff.bone_shield.remains<gcd.max*2
-actions.high_prio_actions+=/death_strike,if=buff.coagulopathy.up&buff.coagulopathy.remains<=gcd.max*2
-actions.high_prio_actions+=/any_dnd,if=!buff.death_and_decay.up
-actions.high_prio_actions+=/blood_boil,if=dot.blood_plague.remains<gcd.max*2
-actions.high_prio_actions+=/soul_reaper,if=active_enemies=1&(target.time_to_pct_35<5)&target.time_to_die>(dot.soul_reaper.remains+5)&(!hero_tree.sanlayn|pet.dancing_rune_weapon.remains<5)
-actions.high_prio_actions+=/rune_tap,use_off_gcd=1,if=rune>3
-
-actions.san_cds=abomination_limb,if=!buff.dancing_rune_weapon.up
-actions.san_cds+=/dancing_rune_weapon
-actions.san_cds+=/bonestorm,if=buff.death_and_decay.up&buff.bone_shield.stack>5&cooldown.dancing_rune_weapon.remains>15
-actions.san_cds+=/tombstone,if=(!buff.dancing_rune_weapon.up&buff.death_and_decay.up)&buff.bone_shield.stack>5&runic_power.deficit>=30&cooldown.dancing_rune_weapon.remains>25
 
-actions.san_drw=bonestorm,if=buff.death_and_decay.up&buff.bone_shield.stack>5
-actions.san_drw+=/death_strike,if=(active_enemies=1|buff.luck_of_the_draw.up)&runic_power.deficit<variable.rp_deficit_threshold
+actions.san_drw+=/heart_strike,if=buff.essence_of_the_blood_queen.remains<1.5&buff.essence_of_the_blood_queen.remains
+actions.san_drw+=/bonestorm,if=buff.bone_shield.stack>=5&buff.death_and_decay.remains
+actions.san_drw+=/death_strike,if=runic_power.deficit<36
 actions.san_drw+=/blood_boil,if=!drw.bp_ticking
+actions.san_drw+=/any_dnd,if=(active_enemies<=3&buff.crimson_scourge.remains)|(active_enemies>3&!buff.death_and_decay.remains)
 actions.san_drw+=/heart_strike
 actions.san_drw+=/death_strike
 actions.san_drw+=/consumption
 actions.san_drw+=/blood_boil
```

Comment:
It's kinda worrying that (as far as I can *barely* see), you potentially have no Marrowrends during San'layn DRW.  Sounds like maybe a DPS practice that will get tanks killed, unless I'm overlooking a change.

Probably need to be sure that any mitigation entries (Icebound Fortitude, Rune Tap) are carried over as well.